internal/packagemanager/packagemanager.go

```diff
@@ -212,6 +197,27 @@ func (pm *DistroPackageManager) GetSSMPackage() artifact.Package {
 	)
 }
 
+func (pm *DistroPackageManager) caCertsPackage() artifact.Package {
+	return artifact.NewPackageSource(
+		artifact.NewCmd(pm.manager, pm.installVerb, "ca-certificates", "-y"),
```

Comment:
nit: define this in constants
```suggestion
		artifact.NewCmd(pm.manager, pm.installVerb, caCertificatesPkgName, "-y"),
```